src/containers/Storage/TableGroup/TableGroup.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import {ArrowToggle, Label, Text} from '@gravity-ui/uikit';
+
+import {cn} from '../../../utils/cn';
+
+import './TableGroup.scss';
+
+const b = cn('ydb-table-group');
+
+interface TableGroupProps {
+    children?: React.ReactNode;
+
+    title: string;
+    entityName: string;
+    count: string | number;
+    expanded?: boolean;
+}
+
+export function TableGroup({
+    children,
+    title,
+    entityName,
+    count,
+    expanded = false,
+}: TableGroupProps) {
+    const [isExpanded, setIsExpanded] = React.useState(false);
+
+    React.useEffect(() => {
+        setIsExpanded(expanded);
+    }, [expanded]);
+
+    const toggleCollapsed = () => {
+        setIsExpanded((value) => !value);
+    };
+
+    const renderTitle = () => {
+        return (
+            <div onClick={toggleCollapsed} className={b('title-wrapper')}>
```

Comment:
Lets try to use button here, otherwise the element isn't reachable from keyboard

Reply:
Button applies it's own styles, that doesn't fit here, so I added `tabIndex` to `div`

Reply:
For now I may focus, but it doesn't expand on Enter.
Lets use button css api, like in this [example](https://github.com/gravity-ui/uikit/blob/b46f53603a260c7699e5bcc5c6899ed8cf13776d/src/components/Button/__stories__/Button.stories.tsx#L203).